pkg/geo/geomfn/azimuth_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package geomfn
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/twpayne/go-geom"
+)
+
+var (
+	zero            = 0.0
+	aQuarterPi      = 0.7853981633974483
+	towQuartersPi   = 1.5707963267948966
+	threeQuartersPi = 2.356194490192344
```

Comment:
do you mind moving these vars into the `TestAzimuth` function? (unless we're planning to re-use them)

Reply:
Done.